--- a/malloc/obstack.c~   2024-10-19 15:55:20.982877484 +0800
+++ a/malloc/obstack.c    2024-10-19 15:55:40.208457165 +0800
@@ -117,7 +117,7 @@
 /* A looong time ago (before 1994, anyway; we're not sure) this global variable
    was used by non-GNU-C macros to avoid multiple evaluation.  The GNU C
    library still exports it because somebody might use it.  */
-struct obstack *_obstack_compat;
+struct obstack *_obstack_compat = NULL;
 compat_symbol (libc, _obstack_compat, _obstack, GLIBC_2_0);
 #  endif
 # endif